### ISSUE TITLE
[Question] NordVPN installed on Ubuntu with Caprover

### ISSUE THREAD

**Issue by a-maggi** [caprover/caprover#2148]
Has anyone been able to make NordVPN work with CapRover? I have an Ubuntu machine with a CapRover container. When I start the VPN on Ubuntu, the NGINX from CapRover becomes unreachable. I’ve already ensured that all the necessary ports for CapRover are allowed, but I still couldn't make it work.

**Comment by alexislefebvre** [comment]
Does another access, like SSH, still works?

It looks like it makes sense that when you connect the server to the VPN, you can't access to it anymore since it then use the VPN?

**Comment by a-maggi** [comment]
Yeah, I had that issue at the beginning, but I got that fixed after I realized that I needed to whitelist the SSH port 22 on the VPN.
